### PR TITLE
feat: add test for ellipse

### PR DESCRIPTION
**Pull request by Mankavelda**
Hello @Sudha247 , I added test for ellipse as requested.
Please review and let me do any necessary corrections

### CODE REVIEW

**Comment by AryanGodara** [comment]
@Mankavelda Please resolve the conflicts so that it can be reviewed

**Comment by Mankavelda** [comment]
Alright @AryanGodara  I will

**Comment by Mankavelda** [comment]
Alright @AryanGodara  I will

**Comment by Mankavelda** [comment]
Alright @AryanGodara  I will

**Comment by Mankavelda** [comment]
> @Mankavelda Please resolve the conflicts so that it can be reviewed

I did @AryanGodara

File: examples/dune [hunk 1]
```diff
@@ -9,36 +9,6 @@
  (libraries graphics))
 
 (executable
- (name concentric_circles)
```
Comment (Sudha247) [inline examples/dune:12]: Why are these removed from the dune file? Could you please check if they're in sync with the latest `main` branch.

**Review by Sudha247 — COMMENTED** [review]
Thank you @Mankavelda! The test looks good to me.

**Comment by kushalpokharel** [comment]
Hi @Mankavelda, I saw that you also added an ellipse in the examples folder which isn't quite related to this issue of adding a test for the ellipse. Can I request you to make this PR about adding an ellipse into the example as I have already made a PR for adding the `ellipse test`. Thank you

File: examples/dune (hunk 1 above)
Comment (Mankavelda) [reply examples/dune:12]: Ok @Sudha247

**Comment by Mankavelda** [comment]
> Thank you @Mankavelda! The test looks good to me.

Thank you @Sudha247 .My local main brnach was not up to date with the remote branch, that is why I had conflicts, but I have fixed that.

**Comment by nikochiko** [comment]
Thanks! @Mankavelda great work!
